build-appdirs.sh

```diff
@@ -10,11 +10,11 @@ fi
 
 rm -rf appimagetool.AppDir/ || true
 mkdir -p appimagetool.AppDir/usr/bin
-cp -f build/appimagetool appimagetool.AppDir/usr/bin
+cp -f build/out/usr/bin/appimagetool appimagetool.AppDir/usr/bin
 
 cp resources/AppRun appimagetool.AppDir/
-cp build/appimagetool appimagetool.AppDir/usr/bin/
-cp build/mksquashfs appimagetool.AppDir/usr/bin/
+cp build/out/usr/bin/appimagetool appimagetool.AppDir/usr/bin/
```

Comment:
Maybe call it `appimagetool.AppDir` instead of `out` if that is what it is?
`out` so far was meant as the directory in which the final end products are transferred "out" of Docker into the Travis CI host instance for signing and uploading.